### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by webrouse**
Changes:
- Update `db-*` packages to versions with fixed bugs.

### CODE REVIEW

File: src/Connection/HiveOdbcResult.php [hunk 1]
```diff
@@ -34,9 +35,9 @@ public function getResultColumns(): array
 
             // Don't convert date types to object
             if ($nativeType === 'BINARY') {
-                $type = Dibi\Type::BINARY;
+                $type = Type::BINARY;
             } else {
-                $type = Dibi\Type::TEXT;
+                $type = Type::TEXT;
```
Comment (webrouse) [inline src/Connection/HiveOdbcResult.php:40]: Nova verzia coding standards, bolo to potrebne upravit.

File: tests/functional/incremental-date/expected/data/out/tables/in.c-main.incremental.csv.manifest [hunk 2]
```diff
@@ -1 +1 @@
...
...,"value":"date_col"},{"key":"KBC.sanitizedName","value":"date_col"},{"key":"KBC.primaryKey","value":false},{"key":"KBC.uniqueKey","value":false}],"float_col":[{"key":"KBC.datatype.type","value":"FLOAT"},{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"FLOAT"},{"key":"KBC.sourceName","value":"float_col"},{"key":"KBC.sanitizedName","value":"float_col"},{"key":"KBC.primaryKey","value":false},{"key":"KBC.uniqueKey","value":false}],"double_col":[{"key":"KBC.datatype.type","value":"FLOAT"},{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"FLOAT"},{"key":"KBC.sourceName","value":"double_col"},{"key":"KBC.sanitizedName","value":"double_col"},{"key":"KBC.primaryKey","value":false},{"key":"KBC.uniqueKey","value":false}],"string_col":[{"key":"KBC.datatype.type","value":"STRING"},{"key":"KBC.datatype.nullable","value":true},{"key":"KBC.datatype.basetype","value":"STRING"},{"key":"KBC.sourceName","value":"string_col"},{"key":"KBC.sanitizedName","value":"string_col"},{"key":"KBC.primaryKey","value":false},{"key":"KBC.uniqueKey","value":false}]},"columns":["id","timestamp_col","date_col","float_col","double_col","string_col"]}
```
Comment (webrouse) [inline tests/functional/incremental-date/expected/data/out/tables/in.c-main.incremental.csv.manifest:1]: Pribudlo `KBC.uniqueKey` lebo sa aktualizova balicek.

File: README.md [hunk 3]
```diff
@@ -36,7 +36,7 @@ The configuration `config.json` contains following properties in `parameters` ke
     - `schema` - string (required)
 - `columns` - array (optional): List of columns to export (default all columns)
 - `outputTable` - string (required): Name of the output table 
-- `incremental` - bool (optional):  Enables [Incremental Fetching](https://help.keboola.com/components/extractors/database/#incremental-fetching)
+- `incremental` - bool (optional):  Enables [Incremental Loading](https://developers.keboola.com/extend/generic-extractor/incremental)
```
Comment (ondrajodas) [inline README.md:39]: tady by bylo lepší odkazovat na:
https://help.keboola.com/storage/tables/#incremental-loading

Comment (webrouse) [reply README.md:39]: diki,upravim to

**Pull request merged**